client/request_test.go

```
@@ -282,6 +317,40 @@
+func Benchmark_Request_Params(b *testing.B) {
+	req := AcquireRequest()
+	req.AddParams(map[string][]string{
+		"foo": {"bar", "fiber"},
+		"bar": {"foo"},
+	})
+
+	b.ResetTimer()
+	b.ReportAllocs()
+
+	for i := 0; i < b.N; i++ {
+		for range req.Params() {
```

golangci-lint (1.62.2)

[warning] empty-block: this block is empty, you can remove it

(revive)